http/headers/Feature-Policy.json

```diff
@@ -509,6 +509,43 @@
             }
           }
         },
+        "idle-detection": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/idle-detection",
+            "spec_url": "https://wicg.github.io/idle-detection/#api-permissions-policy",
+            "support": {
+              "chrome": {
+                "version_added": "94"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "94",
+                "version_removed": "96"
```

Comment:
Where did this `version_removed` come from?  Could you link the source for this?

Reply:
You're the source for this: https://github.com/mdn/browser-compat-data/pull/16222#issuecomment-1128636469

Reply:
Ah right!  Thanks for the link!